cscs-checks/microbenchmarks/cpu/latency/latency.py

```diff
@@ -68,8 +74,6 @@ class CPULatencyTest(rfm.RegressionTest):
     def set_modules(self):
         if self.current_system.name in {'daint', 'dom'}:
             self.modules = ['craype-hugepages1G']
```

Comment:
```suggestion
        if self.current_system.name in {'daint', 'dom', 'eiger', 'pilatus'}:
            self.modules = ['craype-hugepages1G']
```

Reply:
Thanks a lot @sebkelle1, it would help a lot having this additional test when looking for issues on suspect nodes. Do you also need to add `eiger` and `pilatus` here?

Reply:
there's no effect on `eiger` or `pilatus`, also not with 8M pages